### DISCUSSION TITLE
Eric Simons - StackBlitz

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#241] [Announcements]
- https://twitter.com/ericsimons40
- https://twitter.com/StackBlitz
- https://stackblitz.com/@EricSimons

## Questions

- [x] You once lived at AOL and worked on your own startups. Can you share that story?
- [x] Literally the only job you've ever had is CEO. have you always wanted to build startups? What are some you failed at?

### StackBlitz

- [x] What problem were you trying to solve when you and your cofounder built out StackBlitz?
- [x] What sets's stackblitz apart from other services like GitHub codespaces and gitpod?
- [x] What languages does it support?
- [x] What are web containers and what do you use them for?
- [x] Does it use WASM? What are some cool properties of that?
- [x] Being able to debug and run node of the front end is so cool. Just adding a debugger statement and it working such nice DX!
- [x] From the numbers I've seen it's pretty cheap to run everyone's code, how have you kept those costs down?
- [x] Where has stackblitz had the most success?
- [x] How does stack blitz interact with the open source community? What projects are you excited about?

### Future

- [x] What's next for StackBLitz?
- [x] What's your spiciest dev take?
- [x] What do you think the future of coding looks like?

**Comment by EricSimons** [comment]
>  You once lived at Yahoo and worked on your own startups. Can you share that story?
It was AOL not Yahoo fyi!

My tooltips:
- The new angular.dev site (!)
- viteconf.org
- remix.run

**Comment by just-be-dev** [comment]
Tooltips:
- Blob APIs from [netlify](https://www.netlify.com/blog/introducing-netlify-blobs-beta/) and [val.town](https://twitter.com/stevekrouse/status/1724792828310741045)
- https://github.com/immich-app/immich

**Comment by hipstersmoothie** [comment]
- https://github.com/webdiscus/html-bundler-webpack-plugin
- https://next-video.dev/
